Lib/test/test_zipimport.py

```diff
@@ -804,6 +804,7 @@ def testEmptyFile(self):
         os_helper.create_empty_file(TESTMOD)
         self.assertZipFailure(TESTMOD)
 
+    @unittest.skipIf(support.is_wasi, "mode 000 not supported.")
```

Comment:
Is there some alternative we can apply to get the same test coverage?

Reply:
Not yet. Future version of WASI will have a ``set_permissions`` call to adjust read, write, and exec bit of a file.